### ISSUE TITLE
Put 2nd table in advanced section ("show more") for mainnet/testnet so they look the same

### ISSUE THREAD

**Issue by mflaxman** [merland/seedpicker#12]
That the info for testnet/mainnet is displayed differently is going to confuse normal users trying to practice on testnet, which is something that is highly recommended before creating a multisig scheme.

<img width="1270" alt="Screen Shot 2019-09-27 at 9 11 21 AM" src="https://user-images.githubusercontent.com/448644/65784445-e46e5480-e106-11e9-9768-51c663979cb7.png">

**Comment by merland** [comment]
Agreed. I wasn't really happy about out how this turned out either. Will try to rework the layout to make it less confusing. 
I do have some doubts about the merits regular bitcoin users using testnet. Personally, I think it is better to experiment on mainnet, but with tiny amounts. Testnet sort of introduces another dimension - where some things are the same and some are different - into the learning process and I don't think that is a good thing for a newbie.

**Comment by mflaxman** [comment]
As fees increase testnet will make even more sense in the future. I've also found anecdotally that when you tell people to experiment with real bitcoin they are far more cautious at every step, which limits their experimentation/understanding and sometimes even prevents them from even getting started. On testnet, they'll do far more tests (like using different devices/configs, wiping them, testing receiving, sending, sweeping, etc), which results in much more learning and ultimate security in whatever configuration they settle on.

In other words, it may not be the default use case, but I think testnet support is essential in any security product. Some other UI ideas (very open to others):
1. Make a toggle selector somewhere on the page for network (bitcoin mainnet/testnet) that defaults to mainnet. Bonus points if there's a way to link to it like http://seedpicker.net/#testnet.
2. Create an entirely separate subdomain for testnet, something like http://testnet.seedpicker.net/.

**Comment by merland** [comment]
Alright, let's keep testnet in. 
Your UI ideas are very similar to some that we have discussed before, we'll do something along those lines.

**Comment by merland** [comment]
Currently deployed version separates mainnet from testnet like so: 
[http://seedpicker.net/calculator/last-word.html](http://seedpicker.net/calculator/last-word.html)
[http://seedpicker.net/calculator/last-word.html?network=testnet](http://seedpicker.net/calculator/last-word.html?network=testnet)

The two versions are basically the same, but the displayed xpub formats are selected based on the network. In the main view the most important format is displayed, which is Zpub for mainnet and Vpub for testnet.

**Comment by mflaxman** [comment]
Amazing! I'm closing this issue as this feels resolved to me (please reopen if you disagree).

Great job responding to all these github issues, I think you've built something truly useful to bitcoin (and that's quite rate).
